README.md

```diff
@@ -9,7 +9,7 @@ Before you begin creating an application with this `devfile` code sample, it's h
 
 1. The `devfile.yaml` file has an [`image-build` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L24-L30) that points to your `Dockerfile`.
 2. The [`docker/Dockerfile`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/docker/Dockerfile) contains the instructions you need to build the code sample as a container image.
-3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L31-L44) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
+3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L31-L43) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
 4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L46-L59) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```

Comment:
```suggestion
4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L51-L59) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```

Reply:
On this suggestion is this because in the commands section it points to `build-image` and `deployk8s`? I think I misunderstood then and included the definition of those two commands. I'll have to make that change on all the PRs then if so.

Reply:
> On this suggestion is this because in the commands section it points to `build-image` and `deployk8s`? I think I misunderstood then and included the definition of those two commands. I'll have to make that change on all the PRs then if so.

The one command in the correction, `deploy`, points to both of those as a `composite` command. I am ok if the link points to all of the commands referenced `deploy` including the parent or just `deploy`.

@Jdubrick What is your preference here? Which ever one is chosen will need to be updated on all of your PRs to make it consistent.

Reply:
I think just having it point to `deploy` and not the rest of the items, so your changes should be accepted. I think it makes the most sense and still clearly outlines what the description says it points to. 

I will have to update this on all my PRs so I can shoot you a message once it's done so you don't waste your time right now.

@michael-valdron

Reply:
> I think just having it point to `deploy` and not the rest of the items, so your changes should be accepted. I think it makes the most sense and still clearly outlines what the description says it points to.

:+1: 

> I will have to update this on all my PRs so I can shoot you a message once it's done so you don't waste your time right now.

@Jdubrick No worries, just let me know when you're ready for me to re-review the PRs.